components/calendar/stories/template.js

```diff
@@ -265,7 +263,10 @@ export const Template = ({
 				[`${rootClass}--padded`]: padded,
 				...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
 			})}
-			style=${styleMap(customStyles)}
+			style=${styleMap({
+				"--mod-actionbutton-icon-size": "10px",
```

Comment:
```suggestion
				"--mod-actionbutton-icon-size": "var(--spectrum-chevron-icon-size-100)",
```
I just was looking at this that moved from the default args, because it seemed strange. This modification uses the correct size at large scale as well.

I'm wondering if we need a followup Jira issue to look at this, if our current Storybook is relying on inline styles for it to display correctly. And to look at why it has a UI icon within Action button which uses workflow icons and workflow icon sizes/styles.

Reply:
I can't speak to why the override was there in the first place but the move has to do with the customStyles object being undefined vs. defaulting to an empty object.  I updated it to default to an empty object which results in any properties defined as a default in the args for the Template no longer pulling through.

Reply:
I agree though that this isn't the PR to make adjustments to what the value is